From 4550fb0d3190e2b1c61aed3972889f0645e85df0 Mon Sep 17 00:00:00 2001
From: Dridi Boukelmoune <dridi.boukelmoune@gmail.com>
Date: Sat, 23 Mar 2019 13:51:15 +0100
Subject: [PATCH] Link ncurses libs in libnumatop.la

On systems using the linker option --as-needed like Fedora it fails to build.
---
 Makefile.am | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile.am b/Makefile.am
index 643704a..ee57e7c 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -13,6 +13,7 @@ AM_CFLAGS = -fPIC -O2 -g -Wall -W -Wformat-security -D_FORTIFY_SOURCE=2 -fno-com
 ACLOCAL_AMFLAGS = -I m4
 
 noinst_LTLIBRARIES = libnumatop.la
+libnumatop_la_LIBADD = $(NCURSES_LIBS)
 libnumatop_la_SOURCES = \
 	common/include/os/linux/perf_event.h \
 	common/include/os/map.h \
